monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/home/layout/Header.js

```diff
@@ -0,0 +1,133 @@
+import MenuIcon from '@material-ui/icons/Menu';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Toolbar from '@material-ui/core/Toolbar';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import NodeFilter from '../NodeFilter';
+import GroupSelector from '../GroupSelector';
+import { categories, getIdFromRoute } from './NavigatorLinks';
+
+const lightColor = 'rgba(255, 255, 255, 0.7)';
+
+const styles = (theme) => ({
+  secondaryBar: {
+    zIndex: 0,
+  },
+  menuButton: {
+    marginLeft: -theme.spacing(1),
+  },
+  iconButtonAvatar: {
+    padding: 4,
+  },
+  link: {
+    textDecoration: 'none',
+    color: lightColor,
+    '&:hover': {
+      color: theme.palette.common.white,
+    },
+  },
+  button: {
+    borderColor: lightColor,
+  },
+});
+
+function Header(props) {
+  const { classes, onDrawerToggle } = props;
+  const location = useLocation();
+  const [selected, setSelected] =useState(categories[0].children[0]);
+  useEffect(() => {
+    setSelected(getIdFromRoute(location.pathname));
+  }, [location]);
+  return (
+    <React.Fragment>
+      <AppBar color="primary" position="sticky" elevation={0}>
+        <Toolbar>
+          <Grid container spacing={1} alignItems="center">
+            <Hidden smUp>
+              <Grid item>
+                <IconButton
+                  color="inherit"
+                  aria-label="open drawer"
+                  onClick={onDrawerToggle}
+                  className={classes.menuButton}
+                >
+                  <MenuIcon />
+                </IconButton>
+              </Grid>
+            </Hidden>
+            <GroupSelector />
+            <NodeFilter />
+            <Grid item xs />
+            <Grid item>
+              <Link className={classes.link} href="#" variant="body2">
+                Go to docs
+              </Link>
+            </Grid>
+            <Grid item>
+              <Tooltip title="Alerts • No alerts">
+                <IconButton color="inherit">
+                  <NotificationsIcon />
+                </IconButton>
+              </Tooltip>
+            </Grid>
+            <Grid item>
+              <IconButton color="inherit" className={classes.iconButtonAvatar}>
+                <Avatar src="/static/images/avatar/1.jpg" alt="My Avatar" />
+              </IconButton>
+            </Grid>
+          </Grid>
+        </Toolbar>
+      </AppBar>
+      <AppBar
+        component="div"
+        className={classes.secondaryBar}
+        color="primary"
+        position="static"
+        elevation={0}
+      >
+        <Toolbar>
+          <Grid container alignItems="center" spacing={1}>
+            <Grid item xs>
+              <Typography color="inherit" variant="h5" component="h1">
+                {selected.id}
```

Comment:
<img width="1678" alt="Screenshot 2021-02-16 at 10 09 28" src="https://user-images.githubusercontent.com/38608578/108018925-0f7c7f80-703f-11eb-8d12-2fad49362822.png">

Home page heading should be changed to "Nodes"